server/paymentProviders/transferwise/index.ts

```diff
@@ -54,17 +64,17 @@ async function quoteExpense(connectedAccount, payoutMethod, expense): Promise<Qu
   const quote = await transferwise.createQuote(token, {
     profileId: connectedAccount.data.id,
     sourceCurrency: expense.currency,
-    targetCurrency: payoutMethod.data.currency,
+    targetCurrency: <string>payoutMethod.unfilteredData.currency,
```

Comment:
@kewitz There are a few places where I replaced `data` by `unfilteredData`. My understanding is that `data` will never contain the currency because it's filtered out in https://github.com/opencollective/opencollective-api/blob/a23c49e9a1557fb214e512e7692748b183beb813/server/models/PayoutMethod.ts#L56. 

Let me know if this is a mistake.

Reply:
@Betree makes sense, but I think we can safely expose currency in data to make it simpler.